app/routes/administrative-units/administrative-unit/sites/site/edit.js

```diff
@@ -0,0 +1,21 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AdministrativeUnitsAdministrativeUnitSitesSiteEditRoute extends Route {
+  @service store;
+
+  async model() {
+    let adminUnitSite = this.modelFor(
```

Comment:
This was a bit confusing. I assumed it was the site record while it actually is the model of the parent. Returning the object directly would be clearer I think. `this.model.adminUnitSite.site` -> `this.model.site`

Reply:
But I also need the administrative unit model.

Reply:
Yea but that's not an issue right? I just mean that instead of doing:

```js
return { adminUnitSite }
```

You could do:
```js
return adminUnitSite
```
To remove the extra object wrapper.

Reply:
I'm a bit slow today. 😬